backend/user-service/src/test/python/test.py

```
@@ -0,0 +1,27 @@
+    "password": "xx123123123xx",
+    "email": "landric6124si@raf.rs",
+}
+res = requests.post("http://localhost:8081/api/v1/users", json=reg)
+print(f"Response is 201? {res.status_code == 201}")
+print(res.json())
+
+login = {
+    "username": "stiboly",
+    "password": "xx123123123xx"
+}
+res = requests.post("http://localhost:8081/api/v1/users/login", json=login)
```

Ruff (0.15.9)

[error] Probable use of `requests` call without timeout

(S113)